lib/ui/hooks.dart

```diff
@@ -266,6 +267,22 @@ void _invoke1<A>(void callback(A a)?, Zone zone, A arg) {
   }
 }
 
+/// Invokes [callback] inside the given [zone] passing it [arg1] and [arg2].
```

Comment:
nit: I'd note arity 2 inside the docstring to immediately communicate the meaning of 2 in the name.

Reply:
added